universe/assets_3d/api_assets_3d.go

```diff
@@ -393,3 +394,61 @@ func (a *Assets3d) apiRemoveAsset3dByID(c *gin.Context) {
 
 	c.JSON(http.StatusOK, nil)
 }
+
+// @Summary Update 3d asset meta by its id
+// @Schemes
+// @Description Update 3d asset meta by its id
+// @Tags assets3d
+// @Accept json
+// @Produce json
```

Comment:
You're missing path parameters in the documentation

Reply:
Right, all EPs in this group missing path params
Added them